### ISSUE TITLE
/usr/bin/steam: error while loading shared libraries: /usr/lib64/libc.so.6: cannot allocate memory in static TLS block

### ISSUE THREAD

**Issue by romulasry** [clearlinux/distribution#1613]
Bug?

**Comment by puneetse** [comment]
Not quite. The `/usr/bin/steam` included in the games bundle is just a wrapper to help from [linux-steam-integration](https://github.com/solus-project/linux-steam-integration) but steam still needs to be installed manually. One example of how to do that is described in this Phoronix article: https://www.phoronix.com/scan.php?page=news_item&px=How-To-Steam-Clear-Linux

**Comment by ms178** [comment]
A more user friendly method than the one described on Phoronix is to install Steam through the software center and as a flatpack. It also shows some errors at startup which can be safely ignored.
